src/main/java/com/google/sps/recommendation/ProcessNearestNeighbors.java

```diff
@@ -20,29 +20,7 @@
 import java.util.stream.IntStream;
 import com.google.sps.tool.ArrayUtil;
 
-/* This function of this file is to ONLY pre-process the nearest neigbors of 300,000 words.
- * It will create/update a binary .db file. Time results and optimal batch size
- * are based on running this file on a Google Pixelbook. Time result may change for
- * different hardwares.
- *
- * Processing Time:
- * On a Google Pixelbook, a batch size of 500 words results in the fastest runtime for
- * computing the nearest neighbors, while avoiding OOMs. We may see some speedup by
- * migrating from N4dj matrix operations to a numerical computing library like
- * numpy.
- *
- * Time example:
- * Each batch of 500 words takes about 60,000 - 80,000 ms.
- * Processing 100 batches can take from 1 hour 30 minutes - 2 hours.
- *
- * Some underlying reasons for this slow time performance:
- * 1) N4dj's behind the hood operations. For example, to slice a large matrix by a range of indices,
- *    N4dj has to make a copy which is really slow, especially with a big matrix.
- * 2) Each computation of a batch results in sorting the
- *    cosine similarity values in each row of size 300,000.
- * 3) There is some time factor with serializing the result into the .db file.
- *    Note: We use MapDB API to serialize a HashMap Object, which utilizes Random Access Files (RAF)
- */
+// See NEARESTNEIGHBORINFO.md for more information about this file 
```

Comment:
I would keep the first few sentences in this file:

This function of ProcessNearestNeighbors.java is to only pre-process
the nearest neigbors of a given Word2Vec model. In this application,
a Word2Vec model of 300,000 words was used. The file will create/update
a binary .db file

Reply:
done